components/options/about/about.js

```diff
@@ -0,0 +1,49 @@
+/*
+*    StatusPilatus: Monitor your PC like never before!
+*    Copyright (C) 2018 PilatusDevs
```

Comment:
You made this new file and yet you have a license saying 2018...

Reply:
You got me there